src/cljs/rems/application.cljs

```diff
@@ -630,7 +630,9 @@
                        (when-let [mail (:email attributes)]
                          [info-field (text :t.applicant-info/email) mail {:inline? true}])
                        (when-let [organizations (seq (:organizations attributes))]
-                         [info-field (text :t.applicant-info/organization) (str/join ", " (map organization-name-if-known organizations)) {:inline? true}])]
+                         [info-field (text :t.applicant-info/organization) (str/join ", " (map organization-name-if-known organizations)) {:inline? true}])
+                       (when (#{:so :system} (:researcher-status-by attributes))
+                         [info-field (text :t.applicant-info/researcher-status) [readonly-checkbox {:value true}] {:inline? true}])]
```

Comment:
I wonder if this should be shown as false when it does not exist? This status is kind of important for the REMS instance running the bona fide stuff so it may be better to explicitly show false?

Reply:
Yeah I thought about that, but then it would show an uninteresting field for other installations. I guess I could make it configurable though...

Reply:
We can always add it later. We can have a user experience check when the feature is complete.